### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by ssanderson**

### CODE REVIEW

File: pyfolio/ipycompat.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import IPython
```
Comment (ahgnaw) [inline pyfolio/ipycompat.py]: Could we put this in utils.py?

**Comment by ahgnaw** [comment]
Pending tests, LGTM

File: pyfolio/ipycompat.py (hunk 1 above)
Comment (ssanderson) [inline pyfolio/ipycompat.py]: Talked about this in person.  If we add a `utils` directory, I could see moving this there.  I don't think this belongs in single `utils` file.

**Comment by twiecki** [comment]
Thanks!

**Pull request merged**